tools.go

```diff
@@ -0,0 +1,14 @@
+// +build tools
+
+// Copyright 2019 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package tools
+
+// See build/install-deps.sh for a list of required tools
```

Comment:
NIT: `build/install-deps.sh` is deleted.

Reply:
Fixed in latest updates.